### ISSUE TITLE
Why is datasets constrained to 2.4.0 ?

### ISSUE THREAD

**Issue by Alex-S-H-P** [QData/TextAttack#736]
Hi, my project requires pathos and text attack. Unfortunately, both are incompatible as of right now, since they require different versions of dill.

Underversioning would not work in my case, since I use pretty recent functionalities and have other conflicting dependencies when I try.

I have found, in the dependency chain, that your package requires datasets, from hugging face, to be of version 2.4.0. Newer versions of the package would solve this issue for me, since the last version of datasets can be manually installed in my project.

Is there a functionality that breaks when updating this dependency?

**Comment by jxmorris12** [comment]
No, we're just behind. Feel free to open a PR to upgrade datasets (and transformers?) versions :)

**Comment by qiyanjun** [comment]
`requirements.txt` now pins `datasets>=2.4.0` rather than an exact `==2.4.0`, so the constraint this issue was asking about has since been loosened. Closing — if you're still hitting a specific version conflict, please open a new issue with the versions involved.
